engine/table/src/main/java/io/deephaven/engine/table/impl/by/PartitionByChunkedOperator.java

```diff
@@ -139,11 +144,13 @@ public interface AttributeCopier {
             @NotNull final String... keyColumnNames) {
         this.parentTable = parentTable;
         this.resultName = resultName;
+        this.initialized = false;
```

Comment:
Strictly personal opinion: `this.` is misleading when used unnecessarily (that is, in cases where the assignment unambiguous).
```suggestion
        initialized = false;
```